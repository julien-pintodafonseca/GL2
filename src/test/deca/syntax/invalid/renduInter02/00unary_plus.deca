// Description:
//      Programme lexicalement incorrect
//      => L'opÃ©rateur "+" est un opÃ©rateur binaire
//
// Resultats:
//      Erreur syntaxique
//      Ligne 10 : token '+' inattendu

{
    int x = +7;
    println("x=", x);
}