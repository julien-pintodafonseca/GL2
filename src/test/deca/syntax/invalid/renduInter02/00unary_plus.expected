00unary_plus.deca:10:12: extraneous input '+'
